tests/functional/monitoring/prometheus/metrics/test_ocs_exporter_ceph_credentials.py

```
@@ -0,0 +1,192 @@
+                if mon_line and "allow *" in mon_line[0]:
+                    msg = (
+                        f"Check 5 WARNING: {dedicated_user} has 'allow *' "
+                        f"on mon — may be overly permissive"
+                    )
+                    logger.warning(msg)
+                    soft_failures.append(msg)
+            logger.info(
+                "Check 5 PASSED: %s has scoped Ceph permissions",
+                dedicated_user,
+            )
+        except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)